exchanges/binanceus/binanceus.go

```diff
@@ -150,11 +150,11 @@ var (
 // GetServerTime this endpoint returns the exchange server time.
 func (bi *Binanceus) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, error) {
 	var response ServerTime
-	return response.Timestamp,
-		bi.SendHTTPRequest(ctx,
-			exchange.RestSpotSupplementary,
-			serverTime, spotDefaultRate,
-			&response)
+	err := bi.SendHTTPRequest(ctx,
+		exchange.RestSpotSupplementary,
+		serverTime, spotDefaultRate,
+		&response)
```

Comment:
```suggestion
	err := bi.SendHTTPRequest(ctx, exchange.RestSpotSupplementary, serverTime, spotDefaultRate, &response)
```

Reply:
🫶

Reply:
We can not use a single line here. Because the return `response.Timestamp.Time()` is what will be processed first and the the `bi.SendHTTp...` thing next. This will leave us with zero value time value.